cppcache/integration-test/CacheHelper.cpp

```diff
@@ -494,26 +494,30 @@ std::shared_ptr<Pool> CacheHelper::createPool2(
 }
 
 void CacheHelper::logPoolAttributes(std::shared_ptr<Pool>& pool) {
+  using namespace apache::geode::util::chrono::duration;
   LOG("logPoolAttributes() entered");
   LOGINFO("CPPTEST: Pool attribtes for pool %s are as follows:",
           pool->getName());
-  LOGINFO("getFreeConnectionTimeout: %d", pool->getFreeConnectionTimeout());
+  LOGINFO("getFreeConnectionTimeout: %s",
+          to_string(pool->getFreeConnectionTimeout()).c_str());
   LOGINFO("getLoadConditioningInterval: %d",
-          pool->getLoadConditioningInterval());
+          to_string(pool->getLoadConditioningInterval()).c_str());
   LOGINFO("getSocketBufferSize: %d", pool->getSocketBufferSize());
-  LOGINFO("getReadTimeout: %d", pool->getReadTimeout());
+  LOGINFO("getReadTimeout: %d", to_string(pool->getReadTimeout()).c_str());
```

Comment:
I think format should be %s on all the ones you changed to pass char*.

Reply:
Yes great catch!